edx_notifications/stores/tests/test_store.py

```diff
@@ -260,8 +267,8 @@ def test_base_methods_exceptions(self):
 
         with self.assertRaises(NotImplementedError):
             bad_provider.get_all_active_timers()
-
-        with self.assertRaises(NotImplementedError):
+        
+         with self.assertRaises(NotImplementedError):
```

Comment:
I think there's an extra space here which will be a pylint/pep8 error (or even a python compile error).